src/main/drivers/flash_m25p16.c

```diff
@@ -308,16 +315,16 @@ void m25p16_pageProgramBegin(uint32_t address)
     m25p16_enable(bus);
 
     spiTransfer(bus->busdev_u.spi.instance, command, NULL, isLargeFlash ? 5 : 4);
-}
 
-void m25p16_pageProgramContinue(const uint8_t *data, int length)
-{
     spiTransfer(bus->busdev_u.spi.instance, data, NULL, length);
+
+    m25p16_disable(bus);
+
+    currentWriteAddress += length;
 }
 
 void m25p16_pageProgramFinish(void)
```

Comment:
Shouldn't we remove `m25p16_pageProgramFinish` and `m25p16_pageProgramBegin` and rename `m25p16_pageProgramContinue` to `m25p16_pageProgram`?

Reply:
flashfs makes heavy use of these functions with some sort of dual buffering; too much hassle to rewrite them together ...